v2/googlecloud-to-elasticsearch/README.md

```diff
@@ -0,0 +1,12 @@
+# Google Cloud to Elasticsearch Dataflow templates
+
```

Comment:
The [GoogleCloudToElasticsearch](/googlecloud-to-elasticsearch)  is the top-level module containing a batch of templates enabling data integration between Google cloud products and Elasticsearch.